frontend/src/component/impact-metrics/ImpactMetricsChart.tsx

```diff
@@ -16,33 +16,38 @@ type ImpactMetricsChartProps = {
     selectedRange: 'hour' | 'day' | 'week' | 'month';
     selectedLabels: Record<string, string[]>;
     beginAtZero: boolean;
+    showRate?: boolean;
```

Comment:
boolean is almost never a good modelling option. I'd go for a union type because the moment you get a third option it's trivial to extend.